db/migrate/20231024174016_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] According to the ERD diagram, ensure that the `user_id` column is called `author_id`. This is easily achieved by doing something like this 👇 
```ruby
t.references :author, foreign_key: { to_table: 'users' }
#### ERD Diagram
![image](https://github.com/Abdulwhbhussain/blog-app/assets/46839548/3b5c3f03-b9ec-4fd2-9eb8-f79c85fad83e)

```

Reply:
class RenameUserIdColumnToAuthorIdColumn < ActiveRecord::Migration[7.1]
  def change
    rename_column :posts, :user_id, :author_id
  end
end

"I did it using this migration code and file."

Reply:
This is not right I used **rename_column** method instead of doing your way.